ci-operator/jobs/redhat-developer/odo/redhat-developer-odo-master-presubmits.yaml

```diff
@@ -1,5 +1,127 @@
 presubmits:
   redhat-developer/odo:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    name: pull-ci-redhat-developer-odo-master-e2e
+    rerun_command: /test e2e
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-cluster-profile
+        - --target=e2e
+        - --template=/usr/local/e2e
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: redhat-developer-odo-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: e2e
+        - name: TEST_COMMAND
+          value: |
+            make configure-installer-tests-cluster
+            export PATH="$PATH:$(pwd)"
+            export CUSTOM_HOMEDIR="/tmp/artifacts"
+            make test-login-e2e
+            make test-json-format-output
+            make test-java-e2e
```

Comment:
Don't use ```test-java-e2e``` as i am seeing some failure while trying it locally. I am kind of sure that same failure will be observed on prow too. 

So remove ```make test-java-e2e``` as of now. Will keep adding one by one after fixing the issue from test side.

Reply:
Ok

Reply:
```make test-login-e2e``` and ```make test-json-format-output``` looks good to me